scripts/extract_download_logs.py

```diff
@@ -57,7 +80,32 @@
     "rp_start",
     "rp_end",
 ]
-OUTPUT_FILENAME = options.filename
+OUTPUT_FILENAME = args.filename
+
+
+def send_notify(
+    from_date: datetime.datetime,
+    to_date: datetime.datetime,
+    file: StringIO,
+    api_key: str = os.environ["NOTIFY_API_KEY"],
+    template_id: str = "196e5553-886c-40bd-ac9a-981a7868301b",
+    email_address: str = os.getenv("email", "test@example.com"),
```

Comment:
"email" is quite a generic env var name, might be better to call it "NOTIFY_SEND_EMAIL" (note capital casing as well)

Reply:
updated in: 8ba1363